File: chapter1/mario/mario3.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Prompt user for the height
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height <= 0);

    // For the height
    for (int i = 0; i < height; i++)
    {
        // Print the block
        printf("#\n");
    }

}

File: chapter5/mario3.py
from cs50 import get_int

def main():
    # Get height from the user
    n = get_height()

    # Print a square of blocks
    for i in range(n):
        for j in range(n):
            print("#", end="")
        print()

def get_height():
    while True:
        n = get_int("Height: ")
        if n > 0:
            break
    return n

main()
